meetings/__init__.py

```diff
@@ -95,37 +122,52 @@ def meeting_id(self, room_id):
     return f"{room_id}-{datetime.today().strftime('%Y-%m-%d')}"
     
   @command.new(aliases=["sm"])
-  async def startmeeting(self, evt: MessageEvent) -> None:
+  @command.argument("meetingname", pass_raw=True)
+  async def startmeeting(self, evt: MessageEvent, meetingname) -> None:
     meeting = await self.meeting_in_progress(evt.room_id)
-    
     if not await self.check_pl(evt):
-      await evt.respond("You do not have permission to start a meeting")
+      await self.send_respond(evt, "You do not have permission to start a meeting", meeting=meeting)
     elif meeting:
-      await evt.respond("Meeting already in progress")
+      await self.send_respond(evt, "Meeting already in progress", meeting=meeting)
     else:
       # Do backend-specific startmeeting things
       await self.backend.startmeeting(self, evt)
       
+      initial_topic = ""
```

Comment:
Should we give this a proper header? "Opening remarks" or something?

Reply:
i might make this none/null or something other than the empty string -- some meetings never set a topic, and i dont think we should force one on them

Reply:
Good point. Empty string works for now, we can roll forward with it